### ISSUE TITLE
Problems about processing Assembly - Joint dataset with new files

### ISSUE THREAD

**Issue by QuanYuhan** [AutodeskAILab/Fusion360GalleryDataset#95]
Hi, thanks for providing such a great dataset and tools. 

For the Assembly - Joint dataset, I would like to know how to generate a json file from a new .step or .smt file for prediction by the JoinABLe model, including
1. For the parts, refer to https://github.com/AutodeskAILab/Fusion360GalleryDataset/issues/92#issuecomment-1171724576, can I generate a networkx json file that can be used for the JoinABLe model by modifying [solid_to_graph.py.](https://github.com/AutodeskAILab/UV-Net/blob/main/process/solid_to_graph.py)?
2. For joint sets, how to generate joint_set_****.json file from CAD file? That is to reverse the process of [joint2cad](https://github.com/AutodeskAILab/Fusion360GalleryDataset/tree/master/tools/joint2cad). 

Looking forward to your reply and thanks for your help!

**Comment by karldd** [comment]
1. I plan to post the code to do this using Fusion 360, see [here](https://github.com/AutodeskAILab/JoinABLe/issues/3#issuecomment-1293845798). But might not be for a week or two.
2. Currently we don't have code to do exactly what you want to. However, using the Fusion 360 API it should be possible.

**Comment by QuanYuhan** [comment]
Thank you for your reply!

Regarding the second question, I found that in the assembly.json file of the Assembly dataset, the "joints" field provides similar information to the "joints" field of the joint_set_****.json of the Assembly - Joint dataset. It seems that they can be generated by similar code and Fusion 360 API. At the same time I found that you mentioned in https://github.com/AutodeskAILab/Fusion360GalleryDataset/issues/93#issuecomment-1279641286 that the code for parsing the .f3d file is not planned to be open sourced. I would like to know if the code that is not open sourced here refers to the code that generates assembly.json of Assembly dataset?

**Comment by karldd** [comment]
Yes the code we don't plan to release is the code that generates the `assembly.json`.
